Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,7 +3,7 @@
   "loc.helpMarkDown": "[Learn more about this task](https://aka.ms/azurefilecopyreadme)",
   "loc.description": "Copy files to Azure Blob Storage or virtual machines",
   "loc.instanceNameFormat": "$(Destination) File Copy",
-  "loc.releaseNotes": "What's new in Version 4.0: <br/>&nbsp;&nbsp; Support AzCopy.exe version 10.3.3",
+  "loc.releaseNotes": "What's new in Version 4.0: <br/>&nbsp;&nbsp; Support AzCopy.exe version 10.8.0",
```

Comment:
This change must be reflected in all different language files of resources.resjson
Try to build the task , it must automatically be reflected then push the changes to this PR